### DISCUSSION TITLE
No longer Receiving Notifications on HomePod or Apple TV

### DISCUSSION THREAD

**Discussion by jwolynia** [dgreif/ring#638] [Q&A] [answered]
At one point, every ring on my ring doorbell chimed on my Homepod and displayed on my Apple tv.  Sometime over the last couple of months, the feature stopped working. After changing various config options and attempting some suggestions, I am puzzled.  Has anyone else experienced this problem or know where to begin to research my issue?

**Comment by dgreif** [comment, marked as answer]
@jwolynia there were some issues with notifications that should be fixed in the latest version.  Would you mind updating and trying again?
